requirements-dev.txt

```diff
@@ -17,7 +16,6 @@ entrypoints==0.3          # via flake8
 flake8==3.7.9             # via -r requirements-dev.in
 freezegun==0.3.15         # via -r requirements-dev.in
 idna==2.9                 # via -c requirements.txt, requests
-importlib-metadata==1.5.0  # via -c requirements.txt, pluggy, pytest
```

Comment:
Which version of python were you using, 3.6 or 3.8?

Reply:
Ah-ha. That was what was causing that. I've switched to 3.6, and most of the changes have gone away. `appnope` is still going away, but I don't know why.

Reply:
Probably because you're using linux... ["appnope: Disable App Nap on OS X 10.9"](https://pypi.org/project/appnope/). Unfortunately pip-tools isn't clever enough to leave it in for you, and there isn't a way to spoof the environment that I know of. I guess I can just run `freeze-requirements` for you. Why on earth does IPython want appnope anyway? 🤔

Reply:
Pushed frozen requirements.

Reply:
That explains that. Thanks.